tests/integration_tests/database_storage/test.py

```diff
@@ -295,39 +258,21 @@ def test_store_run_init_info(simulation):
 
     tinit_info = db.root.initial_depcode_siminfo[0]
 
-    try:
-        assert tinit_info[0] == simulation.sim_depcode.npop
-        assert tinit_info[1] == simulation.sim_depcode.active_cycles
-        assert tinit_info[2] == simulation.sim_depcode.inactive_cycles
-        assert str(tinit_info[3])[2:-1] == init_info['depcode_name']
-        assert str(tinit_info[4])[2:-1] == init_info['depcode_version']
-        assert str(tinit_info[5])[2:-1] == init_info['title']
-        assert str(tinit_info[6])[2:-1] == init_info['depcode_input_filename']
-        assert str(tinit_info[7])[2:-1] == init_info['depcode_working_dir']
-        assert str(tinit_info[8])[2:-1] == init_info['xs_data_path']
-        assert tinit_info[9] == init_info['OMP_threads']
-        assert tinit_info[10] == init_info['MPI_tasks']
-        assert tinit_info[11] == init_info['memory_optimization_mode']
-        assert tinit_info[12] == init_info['depletion_timestep']
-        assert tinit_info[13] == init_info['execution_time']
-        assert tinit_info[14] == init_info['memory_usage']
-    except AssertionError:
-        db.close()
-        os.remove(db_file)
-        simulation.db_path = db_path_old
-        raise AssertionError("incorrect Value")
-    except KeyError:
-        db.close()
-        os.remove(db_file)
-        simulation.db_path = db_path_old
-        raise KeyError("incorrect key")
-    except BaseException:
-        db.close()
-        os.remove(db_file)
-        simulation.db_path = db_path_old
-        print("something went wrong. See error stack for details")
-
-        print("")
+    assert tinit_info[0] == simulation.sim_depcode.npop
+    assert tinit_info[1] == simulation.sim_depcode.active_cycles
+    assert tinit_info[2] == simulation.sim_depcode.inactive_cycles
+    assert str(tinit_info[3])[2:-1] == init_info['depcode_name']
+    assert str(tinit_info[4])[2:-1] == init_info['depcode_version']
+    assert str(tinit_info[5])[2:-1] == init_info['title']
+    assert str(tinit_info[6])[2:-1] == init_info['depcode_input_filename']
+    assert str(tinit_info[7])[2:-1] == init_info['depcode_working_dir']
+    assert str(tinit_info[8])[2:-1] == init_info['xs_data_path']
+    assert tinit_info[9] == init_info['OMP_threads']
+    assert tinit_info[10] == init_info['MPI_tasks']
+    assert tinit_info[11] == init_info['memory_optimization_mode']
+    assert tinit_info[12] == init_info['depletion_timestep']
+    assert tinit_info[13] == init_info['execution_time']
+    assert tinit_info[14] == init_info['memory_usage']
```

Comment:
How come the `try-except` loop is no longer needed here or at line 315-329?

Reply:
It was buggy and didn't really help with the testing, so I removed it.